app/database/operator/server_data_operator/transformers/user.ts

```diff
@@ -39,7 +39,7 @@ export const transformUserRecord = ({action, database, value}: TransformerArgs):
         user.position = raw?.position ?? '';
         user.roles = raw.roles;
         user.username = raw.username;
-        user.notifyProps = raw.notify_props;
```

Comment:
@enahum, As the `thread` participants don't return the complete user object, shall we use `record?.field` as a fallback value for other fields as well? The only downside is .. in case any of the fields should go back to null or undefined, then it would always take the fallback value.

Reply:
What does `notify_props` has to do with it ? can you explain?

Reply:
For the mentions screen, we depend on `notifyProps.mention_keys`. When we load threads, we are overwriting `notifyProps` with an empty value as we are not getting the complete user object.

Reply:
> For the mentions screen, we depend on notifyProps.mention_keys. When we load threads, we are overwriting notifyProps with an empty value as we are not getting the complete user object.

Then the real question is why are wwe overriding the notify_props.. yes of curse the endpoint retuns the sanitized data, but we shoild exclude the current user from updating, as we do in other places

Reply:
Got it ... I will exclude the current user as well then! 🙂